<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MorphologyWidget</class>
 <widget class="QWidget" name="MorphologyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>311</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="species_label">
      <property name="text">
       <string>Вид</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="ExtendedComboBox" name="combo_box_species"/>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="push_button_find">
      <property name="text">
       <string>Найти</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_found">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ExtendedComboBox</class>
   <extends>QComboBox</extends>
   <header>widget.widget_ext_combo_box</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
